invenio_webhooks/views.py

```diff
@@ -91,14 +90,33 @@ class ReceiverEventListResource(MethodView):
     @error_handler
     def post(self, receiver_id=None):
         """Handle POST request."""
+        return self.handle_event('new_task', receiver_id)
+
+    @require_api_auth()
+    @require_oauth_scopes('webhooks:event')
+    @error_handler
+    def head(self, receiver_id=None):
+        """Handle HEAD request."""
+        return self.handle_event('get_status', receiver_id)
```

Comment:
I don't understand this design. Why are you creating new event if someone wants to know it's status? You need new resource with `event_id`.